### PR TITLE
Add recipe for Tinytimer

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for Tinytimer

### CODE REVIEW

File: recipes/tinytimer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - python
+    - pypandoc
+
+test:
+  imports:
+    - tinytimer
+
+about:
+  home: https://github.com/iskandr/microbench
+  license: Apache
```
Comment (synapticarbors) [inline recipes/tinytimer/meta.yaml]: `license: Apache-2.0`

Also include `license_file` manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/tinytimer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  version: "0.0.0"
+
+source:
+  fn: tinytimer-0.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/5c/87/75d3fbe15aabd45d9b70702241787cf1f7f30dd9fabcd9bc89d828c7661d/tinytimer-0.0.0.tar.gz
+  md5: 83d6988d1ca1702fcbd9ba5eae9d4fd7
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (synapticarbors) [inline recipes/tinytimer/meta.yaml]: Should be platform independent, so don't skip. Also consider making `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/tinytimer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package:
+  name: tinytimer
+  version: "0.0.0"
+
+source:
+  fn: tinytimer-0.0.0.tar.gz
+  url: https://files.pythonhosted.org/packages/5c/87/75d3fbe15aabd45d9b70702241787cf1f7f30dd9fabcd9bc89d828c7661d/tinytimer-0.0.0.tar.gz
```
Comment (synapticarbors) [inline recipes/tinytimer/meta.yaml]: use canoncial url and `sha256`. See the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

File: recipes/tinytimer/build.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (synapticarbors) [inline recipes/tinytimer/build.sh]: Build using pip:

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/tinytimer/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pypandoc
+
+  run:
+    - python
+    - pypandoc
```
Comment (synapticarbors) [inline recipes/tinytimer/meta.yaml]: `pypandoc` does not appear to be required to `run` this package.

**Comment by dougalsutherland** [comment]
FYI, we're having issues with converting recipes into feedstocks right now (#5846), so it'll take a bit for this recipe to appear.

**Pull request merged**